src/transformers/models/llama/configuration_llama.py

```diff
@@ -78,7 +78,7 @@ class LlamaConfig(PretrainedConfig):
             End of stream token id.
         pretraining_tp (`int`, *optional*, defaults to 1):
             Experimental feature. Tensor parallelism rank used during pretraining. Please refer to [this
-            document](https://huggingface.co/docs/transformers/parallelism) to understand more about it. This value is
+            document](https://huggingface.co/docs/transformers/main/en/perf_train_gpu_many#tensor-parallelism) to understand more about it. This value is
```

Comment:
nit: for the English docs, we don't need to index to `/en/`

```suggestion
            document](https://huggingface.co/docs/transformers/main/perf_train_gpu_many#tensor-parallelism) to understand more about it. This value is
```